Makefile

```diff
@@ -137,8 +137,8 @@ ifeq ($(PYTHON_VER),python2.7)
 	@$(SYS_PYTHON) $(VIRTUAL_BOOTSTRAP) $(VIRTUALENV_OPTS) --system-site-packages $(BLD_DIR_ENV)
 else ifeq ($(PYTHON_VER),python3.8)
 	@$(SYS_PYTHON) -m pip install --upgrade pip
-	@$(SYS_PIP) install virtualenv
```

Comment:
Curious question: Using virtualenv had worked for Apple M1 Macs for us which are also based on ARM architecture chips. Any idea why this failed for our check here (I'm pretty new to ARM arch)? Is there some difference for ubuntu CircleCI ARM nodes?

Also since we are updating the Makefile and to not have any unwanted regressions for different platform offerings. Can we try using the `venv` way like in an if check (maybe checking for arch==arm64 and os==ubuntu etc I think?) which can help us distinguish between this Linux ARM CI check and with all other ways we can use virtualenv?

Reply:
I am not sure what is the problem with `virtualenv`. For some reason it was creating Python2.x env.
I am not very experienced with Python and the first hit when searching for `Python virtual environment` Google gave me was https://docs.python.org/3/library/venv.html. So I just replaced `virtualenv` with `venv` and it worked.

Do you face any problems with `venv` on Mac ?

Reply:
https://stackoverflow.com/a/64586483/497381 - explains what it the difference between `virtualenv` and `venv`.
Let me try again with `virtualenv`!

Reply:
`virtualenv -p python3.8` works fine!